examples/text-generation/run_generation.py

```diff
@@ -85,6 +87,12 @@ def setup_parser(parser):
         type=str,
         help="Optional argument if you want to assess your model on a given dataset of the HF Hub.",
     )
+    parser.add_argument(
+        "--dataset",
+        default="/mnt/weka/data/mlperf_inference/llama2/processed-data.pkl",
```

Comment:
remove default, and put None, but do a check later

Reply:
same as above